src/collector.ts

```diff
@@ -157,10 +157,10 @@ class GomodDependencyCollector implements IDependencyCollector {
     async collect(contents: string): Promise<Array<IDependency>> {
         let promiseExec = new Promise<Set<string>>((resolve, reject) => {
             const vscodeRootpath = this.manifestFile.replace("file://", "").replace("/go.mod", "")
-            exec(`go list -f '{{ join .Imports "\\n" }}' ./...`,
+            exec(`${get_golang_executable} list -f '{{ join .Imports "\\n" }}' ./...`,
```

Comment:
Could you please add a debug statement which prints get_golang_executable?

Reply:
I have added connection.console.log in server.js instead of this place for golang CA trigger.

Reply:
Isn't get_golang_executable a function? Should be called like get_golang_executable()?